feature-libs/storefinder/styles/components/_store-finder-store-description.scss

```diff
@@ -49,14 +70,22 @@
     text-align: start;
   }
 
-  .cx-feature-value {
```

Comment:
I think we should still keep these `feature-value feature-item` classes for older versions and clean them up later.

Reply:
I've included them in the feature flag